gatsby-node.js

```diff
@@ -84,6 +84,7 @@ exports.onCreateWebpackConfig = ({ actions }) => {
     resolve: {
       modules: [path.resolve(__dirname, 'src'), 'node_modules'],
       alias: {
+        'react-dom': '@hot-loader/react-dom',
```

Comment:
it treats this warning:
![Image 2019-06-02 at 12 07 35 PM](https://user-images.githubusercontent.com/1519448/58760647-8b6d2600-8543-11e9-897f-88c3fd3ce493.png)

from here: https://github.com/gaearon/react-hot-loader/issues/1227#issuecomment-482139583